extensions/lastpass/src/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { Action, ActionPanel, getPreferenceValues, Icon, List, showToast, Toast } from "@raycast/api";
+import { useEffect, useState } from "react";
+import { AccountDetail } from "./accountDetail";
+import { ErrorDetail } from "./errorDetail";
+import { lastPass } from "./utils/cli";
+
+interface Preferences {
+  email: string;
+  password: string;
+}
+
+const faviconForDomain = (url: string) => {
```

Comment:
you can use the [getFavicon](https://developers.raycast.com/utilities/icons/getfavicon) util for this 😸

Reply:
done, thank you ✅